frontend/mr-admin-flate/src/components/notifikasjoner/Varsel.tsx

```diff
@@ -0,0 +1,31 @@
+import { BodyShort, Heading } from "@navikt/ds-react";
+import { BellIcon, ChevronRightIcon } from "@navikt/aksel-icons";
+import styles from "./Varsel.module.scss";
+
+interface IVarsel {
+  tittel: string;
+  melding: string;
+}
+
+export function Varsel({ tittel, melding }: IVarsel) {
```

Comment:
Jeg har ikke brukt Aksel sin egen Alert-komponent fordi skissene i Figma ikke er like, og denne Varsel-komponenten her skal på sikt kunne trykkes på for å gå til en side med notifikasjoner.